### DISCUSSION TITLE
Allow per-app profiles

### DISCUSSION THREAD

**Discussion by EntityReborn** [seerge/g-helper#2258] [Ideas]
An idea would be to switch to a specific CPU / GPU profile when a certain app launches, and revert back to the previous settings after.

**Comment by seerge** [comment]
@EntityReborn hello, i was answering this before here.

Monitoring all processes running in the system is quite resource intensive task, and also you will have to manually enter all those processes somehow and manage all that in UI. It’s against whole idea of the app to be simple and lightweight 

You can already extremely easily switch between profiles in both directions using hotkeys

https://github.com/seerge/g-helper?tab=readme-ov-file#%EF%B8%8F-keybindings
